tests/test_updater_ng.py

```diff
@@ -326,6 +333,69 @@ def test_non_existing_target_file(self) -> None:
         with self.assertRaises(exceptions.DownloadHTTPError):
             self.updater.download_target(info)
 
+    # TAP 14 tests ~ REMOVE COMMENT BEFORE THE NEXT COMMIT!!
+
+    # test case to check for the TAP 14 folder
+    def test_check_folder_tap14(self) -> None:
+        # Creating the parent folder for the TAP 14 folder
+        self.assertTrue(os.path.isdir(self.tap14_directory))
+
+    def test_check_tap14_contents(self) -> None:
+        # Checking specific files inside TAP 14
+        filenames = ["targets.json", "root.json"]
+        for file in filenames:
+            self.assertTrue(
+                os.path.isfile(os.path.join(self.tap14_directory, file))
+            )
+
+        # Checking specific folders inside TAP 14
+        foldernames = ["targets", "1.0.0"]
+        for folder in foldernames:
+            self.assertTrue(
+                os.path.isdir(os.path.join(self.tap14_directory, folder))
+            )
+
+    def test_get_spec_version1(self) -> None:
```

Comment:
Call this `test_get_spec_version_supported` or something. And put the comment below in a docstring.